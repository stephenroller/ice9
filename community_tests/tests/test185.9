# id test (first 3 pass - third fails)			fail

abc09_09;
ZZ09;
a;
_;


;

