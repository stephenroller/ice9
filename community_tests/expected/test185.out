line 6: illegal character (_)
